server/sonar-db-dao/src/main/java/org/sonar/db/component/ComponentDao.java

```diff
@@ -321,6 +326,10 @@ public void update(DbSession session, ComponentUpdateDto component) {
     mapper(session).update(component);
   }
 
+  public void updateTags(DbSession session, ComponentDto component) {
```

Comment:
for me this method is useless : it's already possible to update tags using the the update method, we won't add one method per field as it brings too much noise : we must trust the developper to use the right method and we have unit test to verify use cases.

Let's have a discussion if you disagree.